Units/parser-v.r/v-enum.d/expected.tags

```diff
@@ -0,0 +1,15 @@
+Foo	input.v	/^enum Foo {$/;"	g	line:3	roles:def	end:10
+.aaa	input.v	/^    aaa = 5$/;"	e	line:4	enum:Foo	roles:def
```

Comment:
I wonder why these start from `.`.

Reply:
Because enum values in V always start with a dot when they are referred to. [See here](https://github.com/vlang/v/blob/master/doc/docs.md#enums). Do you think they should be tagged without the dot then? (I'll defer to your wisdom -- I have less experience with how editors/external software will use the tags!)

Reply:
I remembered this af4901def63546ae5ca4d48dae74bbf8d5fa2682 when I read your reply.

Language processors of Fortran emits `foo_` for the name `foo`. `foo` appears in Fortran input file. The language processor emits `foo_` in the object file. `foo_` can be used in the other language like C.

If you think `.aaa` is useful, you can do the same as the Fortran parser.

1. Make a tag for `aaa` by default.
2. Define V specific extra like... `handyEnum` (just an idea)
3. Make a tag for `.aaa` in addition to `aaa` when `handyEnum` extra is enabled.
4. Turn on the `handyEnum` extra by default if you want.

See also #3668.

Reply:
Fixed.